src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/PseRespondToTribunalService.java

```diff
@@ -212,23 +217,36 @@ private String getHearingDate(CaseData caseData, SendNotificationType sendNotifi
      */
     public void sendClaimantEmail(CaseDetails caseDetails) {
         CaseData caseData = caseDetails.getCaseData();
+        boolean isWelsh = isWelsh(caseData);
+        String emailTemplate = isWelsh
+                ? cyNotificationToClaimantTemplateId
+                : notificationToClaimantTemplateId;
         if (YES.equals(caseData.getPseRespondentOrdReqCopyToOtherParty())) {
-            emailService.sendEmail(notificationToClaimantTemplateId,
+            emailService.sendEmail(emailTemplate,
                 caseData.getClaimantType().getClaimantEmailAddress(),
                 buildPersonalisationNotify(caseDetails));
         }
     }
 
     private Map<String, String> buildPersonalisationNotify(CaseDetails caseDetails) {
         CaseData caseData = caseDetails.getCaseData();
+        boolean isWelsh = isWelsh(caseData);
```

Comment:
This is still calling the featureToggleService twice in the same thread. On L220 you have the value of isWelsh. Pass this value to buildPersonalisationNotify on L227 then you don't need to get is value again here.

Reply:
Refactored